packages/dxf-deserializer/index.js

```diff
@@ -0,0 +1,592 @@
+/*
+## License
+
+Copyright (c) 2017 Z3 Development https://github.com/z3dev
+
+All code released under MIT license
+
+*/
+
+const {colorIndex, BYLAYER, getTLA} = require('./autocad')
+const dxf = require('./DxfReader')
+const {instantiateAsciiDxf} = require('./instantiate')
+const translateAsciiDxf = require('./translate')
+
+// //////////////////////////////////////////
+//
+// DXF (Drawing Exchange Format) is a CAD data file format developed by Autodesk
+//
+// //////////////////////////////////////////
+
+function handleError (e) {
+  console.log('error: line ' + e.line + ', column ' + e.column + ', bad character [' + e.c + ']')
+}
+function handleStart (reader, data) {
+  console.log('DXF reader started')
+}
+function handleEnd (reader, data) {
+  console.log('DXF reader completed')
+}
+
+//
+// handle a entity as provided by the reader
+// groups: 0
+// special handling to set defaults as per DXF specifications
+//
+function handleEntity (reader, group, value) {
```

Comment:
insane amount of work ! I cannot even imagine the headaches to deal with the DXF specs!